modules/README.md

```diff
@@ -184,6 +184,8 @@ Pub/Sub subscription. Primarily used for [FSI - MonteCarlo Tutorial][fsi-monteca
   submission of Google Cloud Batch jobs.
 * **[gke-cluster]** ![community-badge] ![experimental-badge] : Creates a
   Kubernetes cluster using GKE.
+* **[pre-existing-gke-cluster]** ![community-badge] ![experimental-badge] : Retrieves an existing
```

Comment:
Ideally this description would display on 1 line when viewed in [this flat list](https://github.com/GoogleCloudPlatform/hpc-toolkit/blob/a9abdadcb5a6dd8ca819851caa6b80e9108622b8/modules/README.md#scheduler).


Maybe something like:
`Retrieves an existing GKE cluster. Substitute for ([gke-cluster]) module.`

That might even need to be shortened.

Reply:
Changes applied. However double/multiple lines are inevitable on smaller screens.